attribute-service-impl/src/test/java/org/hypertrace/core/attribute/service/AttributeServiceImplTest.java

```diff
@@ -461,4 +464,21 @@ private Document createMockDocument(
     when(document.toJson()).thenReturn(sb.toString());
     return document;
   }
+
+  private <T> CloseableIterator<T> convertToCloseableIterator(Iterator<T> iterator) {
```

Comment:
just curious why this was needed - the API didn't change, shouldn't have been a breaking change whether the impl returned a closable or regular iterator?